src/search/indexing/local_import.rs

```diff
@@ -58,20 +56,30 @@ pub async fn index_local(pool: PgPool) -> Result<Vec<UploadSearchMod>, IndexingE
                 ",
                 result.id
             )
-            .fetch_many(&pool)
-            .try_filter_map(|e| async { Ok(e.right().map(|c| c.category)) })
-            .try_collect::<Vec<String>>()
-            .await?;
+                .fetch_many(&pool)
+                .try_filter_map(|e| async { Ok(e.right().map(|c| c.category)) })
+                .try_collect::<Vec<String>>()
+                .await?;
 
             categories.extend(loaders);
 
+            let user = sqlx::query!(
+                "
+                SELECT u.id, u.username FROM users u
+                INNER JOIN team_members tm ON tm.role = 'Owner'
```

Comment:
Don't hardcode the role names, or at least define a system for roles.  Whatever it is, we will want to be consistent with the naming convention.
`Author`, `author`, `Owner`, `owner`, `creator`, etc...
Do we want to use the original creator, or the current owner of the mod?

Reply:
This could just be an enum for now, like `Status`.  It's just better not to hardcode the naming into the SQL.

Reply:
The owner name will always be hardcoded, as it's the only thing the user can't change. All other roles for other users are dynamic, however we need distinction to see who is owner of a mod.

Reply:
If that will be true, then document it somewhere.  It is still a bad idea to inline the owner role into the SQL rather than using a rust constant or an enum.